src/components/appointmentForm/AppointmentForm.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+
+import { ContactPicker } from '../contactPicker/ContactPicker';
+import DatePicker from "react-datepicker";
+
+export const AppointmentForm = ({
+  contacts,
+  addAppointment
+}) => {
+    const initContact = (contacts.length > 0) ? contacts[0].name : '';
```

Comment:
nit: this doesn't need to be a variable.

Reply:
Done